src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx

```diff
@@ -219,6 +228,12 @@ class GlobalSelectionHeader extends React.Component {
     return new Set(this.props.organization.features).has('global-views');
   };
 
+  /**
+   * Identifies if query string has changed (with query params that this component cares about)
+   *
+   *
+   * @return {Boolean|String[]} Returns `false` if did not change, otherwise return an array of params that have changed
```

Comment:
Having one return type would make calling code a bit simpler. Returning a list will make the rest of the component simpler. I usually associated methods starting with `did` as either hook methods or ones that will return a boolean. A name like `changedQueryKeys()` matches a bit better with what the method does. :bike: